datasette/hookspecs.py

```diff
@@ -49,6 +49,11 @@ def extra_template_vars(
     "Extra template variables to be made available to the template - can return dict or callable or awaitable"
 
 
+@hookspec
+def load_template(template, database, table, columns, view_name, request, datasette):
```

Comment:
I think this should use `firstresult=True`: https://docs.pytest.org/en/latest/writing_plugins.html#firstresult